ocaml/xapi/storage_smapiv1_wrapper.ml

```diff
@@ -92,26 +92,8 @@ let indent x = "    " ^ x
 
 let string_of_date x = Date.to_string (Date.of_float x)
 
-(* Sets the logging context based on `dbg`.
-   Also adds a new tracing span, linked to the parent span from `dbg`, if available. *)
 let with_dbg ~name ~dbg f =
-  let open Debuginfo in
-  let di = of_string dbg in
-  Debug.with_thread_associated di.log
-    (fun () ->
-      let name = "SMAPIv1." ^ name in
-      let tracer = Tracing.get_tracer ~name in
-      let span = Tracing.Tracer.start ~tracer ~name ~parent:di.tracing () in
-      match span with
-      | Ok span_context ->
-          let result = f {di with tracing= span_context} in
-          let _ = Tracing.Tracer.finish span_context in
-          result
-      | Error e ->
-          D.warn "Failed to start tracing: %s" (Printexc.to_string e) ;
-          f di
-    )
-    ()
+  Debuginfo.with_dbg ~module_name:"SMAPIv1-Wrapper" ~name ~dbg f
```

Comment:
This changes SMAPIv1 to SMAPIv1-Wrapper, is this an intended change? It's not mentioned in the commit message

Reply:
This was @robhoes's suggestion to be more clear about the module names. As in the next commit https://github.com/xapi-project/xen-api/pull/5293/commits/b69cc0f38e09407e318b931c546d2d413fe2d071, I instrumented 'storage_smapiv1.ml' and put the 'module_name' as 'SMAPIv1'.